doc/manual-install.md

```diff
@@ -225,6 +225,8 @@ npm install
 make
 ```
 
+_NOTE: When installing on older releases keep in mind that you need Node.js >= 10 and npm >= 6._
```

Comment:
Can you change releases for "distributions" here?

Reply:
Changed :)